### PR TITLE
Disable Radamsa + ML RNN for LPM targets

### PR DESCRIPTION
**Pull request by jonathanmetzman**
Radamsa and ML RNN mutate testcases in a corpus.
More often than not, this makes items in the corpus invalid protos.
Since LF/LPM isn't smart enough not to merge these back into the corpus (I think because they reveal new coverage in the parser), the corpus gets (permanently?) gunked up with tescases that LPM can't mutate as a result. This takes time away from real testcases.

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
PTAL.

I suspect it's probably worth doing this for AFL as well since LPM targets are getting built for AFLin OSS-Fuzz.

File: src/python/bot/fuzzers/engine_common.py [hunk 1]
```diff
@@ -154,6 +154,13 @@ def get_merge_timeout(default_merge_timeout):
                                  'MERGE_TIMEOUT_OVERRIDE')
 
 
+def is_lpm_fuzz_target(fuzzer_path):
+  """Returns True if |fuzzer_path| is a libprotobuf-mutator based fuzz
+  target."""
+  with open(fuzzer_path) as fuzzer_handle:
```
Comment (Dor1s) [inline src/python/bot/fuzzers/engine_common.py]: nit: maybe `file_handle` instead of `fuzzer_handle`?

**Comment by mhlakhani** [comment]
Nice catch indeed!

> I suspect it's probably worth doing this for AFL as well since LPM targets are getting built for AFLin OSS-Fuzz.

To add to this - I don't think AFL has support for custom mutators so this problem should definitely exist there unless all the LPM targets are doing something special (and invoking LPM within `LLVMFuzzerTestOneInput`)? We also ran into this issue and had to disable AFL for our equivalent of LPM targets.

**Comment by jonathanmetzman** [comment]
> Nice catch indeed!
> 
> > I suspect it's probably worth doing this for AFL as well since LPM targets are getting built for AFLin OSS-Fuzz.
> 
> To add to this - I don't think AFL has support for custom mutators so this problem should definitely exist there unless all the LPM targets are doing something special (and invoking LPM within `LLVMFuzzerTestOneInput`)? We also ran into this issue and had to disable AFL for our equivalent of LPM targets.

We disabled these on the build side in Chromium, but right now AFL and LPM are running on CF. Actually they've even found bugs. But I'll probably disable this in CF as well.

**Pull request merged**